_author_instructions/04_software_analyses.md

```diff
@@ -1,28 +1,29 @@
 ---
 layout: single
 sidebar:
-  nav: authors_compare_simulations.md
-title: Comparisons of Molecular Simulation Packages
-excerpt: What is a publishable comparison for molecular simulation packages?
-permalink: /authors/compare_simulations/
+  nav: authors_software_analyses.md
+title: Software Analyses
+excerpt: What is a publishable software analyis for molecular simulation packages?
+permalink: /authors/software_analyses/
 ---
 
-## What is a publishable comparison for molecular simulation packages?
+## What is a publishable software analysis paper for molecular simulation packages?
 
-Simulation comparison papers describe attempts to perform the same
-calculation with a range of different simulation programs. 
+Software analysis papers describe attempts to perform the same
+calculation with a range of different simulation programs, perform anaysis of changes in perfomrmance over a series of releases of a single software package.  In some cases, comparisons of the same calculation with different methods may also be appropriate for a software analysis paper.
+ 
 Such comparison should be updated periodically with different versions of
 the same programs (or potentially additional programs).
 
 
-## Additional criteria considered in review of comparisons of molecular simulation programs
+## Additional criteria considered in review of software analysis papers
 * Does the manuscript include developers or advanced users of all programs being compared to ensure that comparisons are fair?
-* Are best practices being used in the simulation comparisons?
+* Are best practices being used in the software analysis?
```

Comment:
What sort of best practices do we mean here?  Perhaps @jppiquem might like to weigh in.  For benchmarking?  Statistical uncertainty?  Let's be as specific as possible or refer to a specific set of standards.

Reply:
I added some suggestions, would be good to get feedback from @jppiquem